polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -84,14 +88,52 @@ bool isCandidate(LoopLikeOpInterface loop) {
   return true;
 }
 
+unsigned getLocalMemRemain(Operation *module) {
+  assert(module->hasTrait<OpTrait::SymbolTable>() &&
+         "Expecting module with SymbolTable trait");
+  unsigned localMemRemain = LocalMemSize;
+  module->walk([&localMemRemain](memref::GlobalOp global) {
+    auto MT = global.getType();
```

Comment:
use camel case vars starting with a lowercase. so MT -> ty, MS -> memSpace

Reply:
also use the static types rather than auto

Reply:
done.